src/modules/flight_mode_manager/FlightModeManager.cpp

```diff
@@ -537,7 +537,7 @@ FlightTaskError FlightModeManager::switchTask(FlightTaskIndex new_task_index)
 	}
 
 	// Save current setpoints for the next FlightTask
-	vehicle_local_position_setpoint_s last_setpoint = FlightTask::empty_setpoint;
+	trajectory_setpoint_s last_setpoint{};
```

Comment:
TODO: review

Reply:
Why not keeping `empty_setpoint`? It should also be used here, otherwise if `!isAnyTaskActive()`, then it will filled with zeros instead of NANs